css/styles.css

```
@@ -78,34 +79,110 @@ img {
 
 /*----------------tonearm------------------*/
 
-.animation_tonearm {
-}
+.animation_tonearm {}
 
-.tonearm {
+.tonearm_wrapper {
+    position: relative;
 }
 
+
+
 .tonearm_main {
+    position: relative;
+    display: block;
+    width: 108px;
+    height: 420px;
+    top: 11px;
+    right: 102px;
+    transform: rotate(0deg);
+    transform-origin: center 90px;
+    z-index: 2;
 }
 
-.tonearm_handler {
+.tonearm_stand {
+    position: absolute;
+    width: 155px;
+    height: 155px;
+    border-radius: 50%;
+    box-shadow: 0px 0px 0px 3px #8A8A8A, inset 0px 4px 4px #F2F2F2;
```

Additional shadow should be added
```suggestion
     box-shadow: 0px 0px 0px 3px #8A8A8A, 0px 0px 3px 6px #000000, inset 0px 4px 4px #F2F2F2;
```